libs/core/src/lib/popover/popover-body/popover-body.component.html

```diff
@@ -10,17 +9,19 @@
     [style.min-width.px]="_popoverBodyMinWidth"
     [style.width.px]="_popoverBodyWidth"
 >
-    <ng-content select="[fd-popover-body-header]"></ng-content>
+    <div fd-scrollbar style="max-height: 100%; max-width: 100%; width: 100%">
```

Comment:
Please don't use inline styles.

Reply:
+1 on this. There shouldn't be inline css